cmd/helm/init.go

```diff
@@ -130,6 +131,7 @@ func newInitCmd(out io.Writer) *cobra.Command {
 	f.BoolVar(&i.opts.EnableHostNetwork, "net-host", false, "install Tiller with net=host")
 	f.StringVar(&i.serviceAccount, "service-account", "", "name of service account")
 	f.IntVar(&i.maxHistory, "history-max", 0, "limit the maximum number of revisions saved per release. Use 0 for no limit.")
+	f.IntVar(&i.replicas, "replicas", 1, "amount of tiller instaces to run on the cluster, increase to get required HA level")
```

Comment:
Would it be possible to remove the second part of this comment, so that it reads "amount of tiller instances to run on the cluster."? Right now, increasing the number of tiller instances won't necessarily make the system more or less HA than it would be with 1 because of a lack of a distributed lock.

Also, there's a typo in this docstring: insta/n/ces